openedx/adg/lms/webinars/helpers.py

```diff
@@ -0,0 +1,22 @@
+"""
+All helper methods for webinars apps
+"""
+from openedx.adg.common.lib.mandrill_client.client import MandrillClient
+from openedx.adg.common.lib.mandrill_client.tasks import task_send_mandrill_email
+
+
+def send_webinar_registration_email(webinar, email):
+    """
+    Send webinar registration email to user.
+    Args:
+        webinar (Webinar): The webinar in which user is registered
+        email (str): User's email
+    Returns:
+        None
+    """
+    task_send_mandrill_email.delay(MandrillClient.WEBINAR_CONFIRMATION, [email], {
+        'webinar_title': webinar.title,
+        'webinar_description': webinar.description,
+        'webinar_link': webinar.meeting_link,
+        'webinar_start_time': webinar.start_time.strftime('%b %d, %Y %I:%M %p GMT'),
```

Comment:
We can get the timezone by `%Z`. Or are we going to hardcode it to GMT?

Reply:
`%Z` will show UTC. UTC and GMT are the same. I couldn't find any way to show GMT instead of UTC